examples/zkapps/02-private-inputs-and-hash-functions/src/main.ts

```diff
@@ -8,53 +8,53 @@ import {
   AccountUpdate,
 } from 'snarkyjs';
 
-(async function main() {
```

Comment:
This is good idea to remove the main functions in these tutorials in favor of using top level awaits. I'm commenting here so we can do this consistently across all the tutorials we update @Trivo25 .

Reply:
@ymekuria What is the oldest NodeJS version that supports top level await? This could affect the decision to pursue this

Reply:
14.3 it seems https://pprathameshmore.medium.com/top-level-await-support-in-node-js-v14-3-0-8af4f4a4d478

Reply:
good enough to get rid of those main functions

Reply:
Agreed.